pkg/registry/core/pod/strategy.go

```diff
@@ -247,17 +247,26 @@ func ResourceLocation(getter ResourceGetter, rt http.RoundTripper, ctx context.C
 	if !valid {
 		return nil, nil, errors.NewBadRequest(fmt.Sprintf("invalid pod request %q", id))
 	}
-	// TODO: if port is not a number but a "(container)/(portname)", do a name lookup.
 
 	pod, err := getPod(getter, ctx, name)
 	if err != nil {
 		return nil, nil, err
 	}
 
 	// Try to figure out a port.
+	parts := strings.Split(port, "/")
+	containerName := ""
+	if len(parts) == 2 {
+		containerName = parts[0]
+		port = parts[1]
+	}
+	// when port is a "(container)/(portname)", do a name lookup.
```

Comment:
Please add a unit test to test the new behavior.

Reply:
Searching under pkg/registry/, I still need to find some sample where ProxyREST#Connect is called.

Reply:
I don't want to add more features to this proxy behavior. I would prefer to just delete the TODO. Is there a strong use case for implementing this?